Example/ViewController.swift

```diff
@@ -24,14 +26,14 @@ class ViewController: AsyncMessagesViewController, ASCollectionDelegate {
         }
         
         let dataSource = DefaultAsyncMessagesCollectionViewDataSource(currentUserID: users[0].ID)
-        super.init(dataSource: dataSource)
+        super.init(dataSource: dataSource)!
```

Comment:
Instead of forcibly unwrapping here, let's make this a failable initializer.

Reply:
Done